web_external/js/views/widgets/ImageViewerWidget.js

```diff
@@ -10,16 +22,40 @@ isic.views.ImageViewerWidget = isic.View.extend({
             this.tileHeight = resp.tileHeight;
             this.sizeX = resp.sizeX;
             this.sizeY = resp.sizeY;
+            this.destroyViewer();
```

Comment:
Don't we want to call `this.destroyViewer();` before setting the tile metadata here, since `destroyViewer` is going to `delete` all the tile metadata if a viewer exists?